webpack/scenes/ContentViews/Details/Filters/CVErrataDateFilterContent.js

```diff
@@ -46,30 +46,37 @@ const CVErrataDateFilterContent = ({
     selectCVFilterDetails(state, cvId, filterId), shallowEqual);
   const { repositories = [], rules } = filterDetails;
   const [{
-    id, types, start_date: ruleStartDate, end_date: ruleEndDate, date_type: ruleDateType,
+    id, types, allow_other_types, start_date: ruleStartDate, end_date: ruleEndDate, date_type: ruleDateType
   } = {}] = rules;
   const { permissions } = details;
   const [startDate, setStartDate] = useState(convertAPIDateToUIFormat(ruleStartDate));
   const [endDate, setEndDate] = useState(convertAPIDateToUIFormat(ruleEndDate));
   const [dateType, setDateType] = useState(ruleDateType);
   const [dateTypeSelectOpen, setDateTypeSelectOpen] = useState(false);
   const [typeSelectOpen, setTypeSelectOpen] = useState(false);
-  const [selectedTypes, setSelectedTypes] = useState(types);
+  const [selectedTypes, setSelectedTypes] = useState(() => {
+    if (allow_other_types) {
+      return[...types, 'other'];
+    }
+    return types;
+  });
```

Comment:
Why not just call `getInitialSelectedTypes()`?

Reply:
Good catch. Changed.